.github/workflows/release.yml

```diff
@@ -24,6 +24,8 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18
+          cache: "pnpm"
+          cache-dependency-path: "pnpm-lock.yaml"
```

Comment:
```suggestion
```

I don't think you need this since its defined in the action [here](https://github.com/actions/setup-node/blob/main/src/cache-utils.ts#L20). I didn't need `cache-dependency-path` [when I got it working](https://github.com/AlexErrant/Pentive/actions/runs/4758458558/workflow) in my own repo:

![chrome_auuC9PQYFY](https://user-images.githubusercontent.com/109672176/233750972-68ff550e-4a43-4d43-806d-2c80f393cf77.png)

Reply:
Ahh yes, makes sense. I found the same in the [codebase](https://github.com/actions/setup-node/blob/5b32c9063c6eb277dacbf3e916a8bc5d2617c5b7/src/cache-utils.ts#L20)

Reply:
Fixed in 76d7e92b080c95faeaec29593db88e7769d26843